src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_engine_test.py

```diff
@@ -118,17 +122,3 @@ def test_fuzz_with_crash(self):
 
     # Testcase (non-zero size) should've been copied back.
     self.assertNotEqual(os.path.getsize(crash.input_path), 0)
-
-  def test_startup_crash_not_reported(self):
```

Comment:
I don't really understand this test. @jonathanmetzman is this still relevant? (it was not being run at all before).

Reply:
I think we don't want AFL to report crashes in corpus. Not 100% sure.